jobs/JEVS_MESOSCALE_STATS

```diff
@@ -54,23 +54,23 @@ export USHevs=${USHevs:-$HOMEevs/ush}
 export SCRIPTSevs=${SCRIPTSevs:-$HOMEevs/scripts}
 export FIXevs=${FIXevs:-$HOMEevs/fix}
 export MASKS=$FIXevs/masks
-export MET_PLUS_CONF="${PARMevs}/metplus_config/mesoscale/grid2obs/stats"
+export MET_PLUS_CONF="${PARMevs}/metplus_config/${STEP}/${COMPONENT}/${VERIF_CASE}"
 export cyc=${cyc:-00}
 export cycle=t${cyc}z
 setpdy.sh
 . $DATA/PDY
 
 if [ $MODELNAME = nam ] || [ $MODELNAME = rap ]; then
   export VDATE=${VDATE:-$PDYm1}
-  export COMIN=${COMIN:-$(compath.py -o $NET/$evs_ver)}
+  export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
   export COMINnam=${COMINnam:-$(compath.py ${envir}/com/nam/${nam_ver})}
   export COMINrap=${COMINrap:-$(compath.py ${envir}/com/rap/${rap_ver})}
   export COMINobsproc=${COMINobsproc:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
   export COMINobs=$COMINobsproc
-  export COMINspcotlk=${COMINspcotlk:-$(compath.py -o $NET/$evs_ver/prep/cam)}
-  export COMINccpa=${COMINccpa:-$(compath.py  ccpa/${ccpa_ver})}
-  export COMINmrms=${COMINmrms:-$DCOMROOT/ldmdata/obs/upperair/mrms}
-  export COMINnohrsc=${COMINnohrsc:-$DCOMROOT}
+  export EVSINspcotlk=${EVSINspcotlk:-$(compath.py -o $COMIN/prep/cam)}
```

Comment:
This can remove compath.py and should be
```
export EVSINspcotlk=${EVSINspcotlk:-$COMIN/prep/cam}
```
I believe have to use cam instead of $COMPONENT b/c this dataset is unique to cam. Is that right, @RoshanShrestha-NOAA ?
Please also remove EVSINspcotlk from the driver scripts.